modules/nw-route-specific-annotations.adoc

```diff
@@ -16,95 +16,121 @@ To create a whitelist with multiple source IPs or subnets, use a space-delimited
-Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks. |
-|`haproxy.router.openshift.io/timeout` | Sets a server-side timeout for the route. (TimeUnits) | `ROUTER_DEFAULT_SERVER_TIMEOUT`
-|`haproxy.router.openshift.io/timeout-tunnel` | This timeout applies to a tunnel connection, for example, WebSocket over cleartext, edge, reencrypt, or passthrough routes. With cleartext, edge, or reencrypt route types, this annotation is applied as a timeout tunnel with the existing timeout value. For the passthrough route types, the annotation takes precedence over any existing timeout value set. | `ROUTER_DEFAULT_TUNNEL_TIMEOUT`
-|`ingresses.config/cluster ingress.operator.openshift.io/hard-stop-after` | You can set either an IngressController or the ingress config . This annotation redeploys the router and configures the HA proxy to emit the haproxy `hard-stop-after` global option, which defines the maximum time allowed to perform a clean soft-stop. | `ROUTER_HARD_STOP_AFTER`
-|`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
+and "-". The default is the hashed internal key name for the route. 
+|
+
+|`haproxy.router.openshift.io/pod-concurrent-connections`
+|Sets the maximum number of connections that are allowed to a backing pod from a router. +
+Note: If there are multiple pods, each can have this many connections.  If you have multiple routers, there is no coordination among them, each may connect this many times. If not set, or set to 0, there is no limit. 
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections`
+|Setting `'true'` or `'TRUE'` enables rate limiting functionality which is implemented through stick-tables on the specific backend per route. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks. 
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.concurrent-tcp`
+|Limits the number of concurrent TCP connections made through the same source IP address. It accepts a numeric value. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+| 
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-http`
+|Limits the rate at which a client with the same source IP address can make HTTP requests. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-tcp`
+|Limits the rate at which a client with the same source IP address can make TCP connections. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+
+|`haproxy.router.openshift.io/timeout` 
+|Sets a server-side timeout for the route. (TimeUnits)
+|`spec.tuningOptions.serverTimeout` 
```

Comment:
I think that mentioning ingresscontroller fields is a good idea as it shows how to set a value for all the routes admitted by the controller. However the `tunningOptions` are not guaranteed to be specified and some default values are set by the router. These default values are often (maybe even always) mentioned in `oc explain`. Do we let the reader find them there or we want the default values to be added to the OCP docs?